gcloud-java-resourcemanager/src/test/java/com/google/gcloud/resourcemanager/ProjectTest.java

```diff
@@ -48,13 +47,19 @@
       .state(STATE)
       .build();
 
+  private ResourceManager serviceMockReturnsOptions = createStrictMock(ResourceManager.class);
+  private ResourceManagerOptions mockOptions = createMock(ResourceManagerOptions.class);
   private ResourceManager resourceManager;
+  private Project expectedProject;
   private Project project;
 
   @Before
   public void setUp() throws Exception {
+    expect(serviceMockReturnsOptions.options()).andReturn(mockOptions).anyTimes();
```

Comment:
I think we should avoid using `anyTimes()` as this makes a lot of different behaviors pass the test. If possible I would prefer a fine grain definition of `expect`, i.e. in each test method rather than in a common `@Before` method. If you think this is not possible we could use [times(min, max)](http://easymock.org/api/org/easymock/IExpectationSetters.html#times-int-int-)